### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by daPhipz**
This adds translations to some untranslated strings and rewords some existing translations to better suit the other translations from the German GNOME translation team.
This is still WIP, because I have two questions (see the comments). Once everything is resolved, I'll mark this PR ready for review ;)

### CODE REVIEW

File: locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po [hunk 1]
```diff
@@ -145,19 +145,18 @@ msgid "Both"
 msgstr "Beides"
 
 #: prefs.js:329
+#, fuzzy
 msgid "Neither"
 msgstr "Nichts"
```
Comment (daPhipz) [inline locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po:150]: "Neither" sounds like there are two or more options the user can choose from. In what context is this string used?

File: locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po [hunk 2]
```diff
@@ -130,7 +130,7 @@ msgstr "Bei Auswahl einfügen"
 #: prefs.js:194
 #, fuzzy
 msgid "Process primary selection"
-msgstr "Bei Auswahl einfügen"
+msgstr "Primäre Auswahl bearbeiten"
```
Comment (daPhipz) [inline locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po:133]: What is the difference between the "primary selection" and a regular selection?

File: locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po (hunk 1 above)
Comment (SUPERCILEX) [reply locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po:150]: https://github.com/SUPERCILEX/gnome-clipboard-history/blob/ba1a28be1a0c7fb7d40ad4d873eaab5a64cc3256/prefs.js#L329

It's for deciding what to show in the top bar: the icon, copied text, both those things, or neither of them (so nothing).

File: locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po (hunk 2 above)
Comment (SUPERCILEX) [reply locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po:133]: https://wiki.archlinux.org/title/clipboard#Selections

Primary just means whatever is in your middle mouse click buffer.

**Comment by SUPERCILEX** [comment]
I merged https://github.com/SUPERCILEX/gnome-clipboard-history/pull/49 because it was opened first, but there are some additional changes here that I'd like to merge too. I'll resolve conflicts later tonight. I'm also going to change the "type to search" thing to "Search clipboard history..." if you want to translate that.

**Comment by daPhipz** [comment]
Thanks for the replies - no worries, I'll wait once you have done the changes and translate the new string!

**Comment by SUPERCILEX** [comment]
@daPhipz Ok, I went ahead and merged stuff into your branch. :+1: Pull and you should be able continue.

**Comment by daPhipz** [comment]
Done - ready to merge as far as I am concerned :)

**Comment by SUPERCILEX** [comment]
Thanks!

**Comment by daPhipz** [comment]
Ummm... I put these NBSP there for a reason 🙈

It is standard procedure in the German translation team for GNOME to put NBSPs like I did - why did you remove them?

**Comment by SUPERCILEX** [comment]
Oh lol, I keep thinking it's a weird quirk of your text editor. Thanks for the explanation! Restored in 7361bd05d47c6ae49584ad8b41a2e1466b5a766d.

**Comment by daPhipz** [comment]
No worries :)
It wouldn't matter anyway for these small strings, but if a longer string would break right before the 3 dots or similar, it would look weird to begin a new line just for those. That's the reasoning behind the nbsp ☺️

**Pull request merged**